src/Sylius/Bundle/AddressingBundle/DependencyInjection/Configuration.php

```diff
@@ -90,6 +92,21 @@ private function addResourcesSection(ArrayNodeDefinition $node)
                                 ->end()
                             ->end()
                         ->end()
+                        ->arrayNode('address_log_entry')
+                            ->addDefaultsIfNotSet()
+                            ->children()
+                                ->variableNode('options')->end()
+                                ->arrayNode('classes')
+                                    ->addDefaultsIfNotSet()
+                                    ->children()
+                                        ->scalarNode('model')->defaultValue(AddressLogEntry::class)->cannotBeEmpty()->end()
+                                        ->scalarNode('controller')->defaultValue(ResourceController::class)->cannotBeEmpty()->end()
+                                        ->scalarNode('repository')->defaultValue(ResourceLogEntryRepository::class)->cannotBeEmpty()->end()
+                                        ->scalarNode('factory')->defaultValue(Factory::class)->end()
```

Comment:
I'm wondering why don't we have `->cannotBeEmpty()` on a factory node also. Any thoughts?